.github/workflows/preview_bump_and_publish.yml

```diff
@@ -10,14 +10,6 @@ jobs:
   preview-bump-and-publish:
     runs-on: ubuntu-latest
     steps:
-      - name: Check branch
-        run: |
-          if [[ "$GITHUB_REF" != "refs/heads/main" && "$GITHUB_REF" != refs/heads/crumbs-* ]]; then
-            echo "This job can only run on the main branch or branches starting with crumbs-"
-            exit 1
-          fi
-        shell: bash
```

Comment:
is it worth removing this? i think the early abort of the ci is quite nice & gives more info - can we tweak the if statement to allow for the newly allowed prefixes?

Reply:
Added the check in `release.config.js`